go/vt/sqlparser/sql.y

```diff
@@ -4496,6 +4496,14 @@ prepare_statement:
   {
     $$ = &Prepare{Name: string($2), Expr: string($4)}
   }
+| PREPARE ID FROM system_variable
```

Comment:
any other reserved keywords we'd maybe need to allow here?

Reply:
none that are reserved, but some that are non_reserved, added those